src/proto/net.proto

```diff
@@ -3,17 +3,31 @@ syntax = "proto2";
 package proto;
 
 message ServerMessage {
+  optional bool status = 1;
+
   oneof message_type {
```

Comment:
I think optional applies to any field, and in the case of oneof allows a completely empty field. I could be wrong, though. It's what people normally do at Google, is what I'm saying, but it makes less sense to require it here.

Also, it is preferred to never change field tags, as it will create incompatibility with old versions of serialized data. Since we are only using these at runtime it's no problem, but if we want to hypothetically keep persistent data or logs then this would be bad.